swssconfig/swssconfig.cpp

```diff
@@ -0,0 +1,203 @@
+#include <stdlib.h>
+#include <iostream>
+#include <vector>
+#include "logger.h"
+#include <fstream>
+#include "dbconnector.h"
+#include "producertable.h"
+#include "json.hpp"
```

Comment:
After discussion, we're still putting the json.hpp file into our common library. it's only one file and better be source controlled from our side to avoid potential future updates breaking the build.